frontend/src/container/AppLayout/index.tsx

```diff
@@ -59,7 +50,7 @@ function AppLayout(props: AppLayoutProps): JSX.Element {
 		(state) => state.app,
 	);
 
-	const [collapsed, setCollapsed] = useState<boolean>(
+	const [collapsed] = useState<boolean>(
```

Comment:
we can remove this as well now ? and can we add the code to clean this localstorage value ? to not pollute it

Reply:
Yes , we can remove all the references and set the value as default.